manon/footer-navigation-link-focus.scss

```diff
@@ -10,9 +10,14 @@ body > footer,
   nav {
     a {
       &:focus,
-      /* Testing purposes */
       &.focus {
         @include link.link-and-icon-styling("footer-navigation-link-focus-");
+
+        &:visited {
+          @include link.link-and-icon-styling(
+            "footer-navigation-link-visited-focus-"
+          );
```

Comment:
Just to be sure, is correct?

- `:focus:visited`
- `:visited:hover`

Reply:
I have used these within the iCore open theme to add the styling as needed. As such I have tested them. 

I've added the variables as well: 
https://github.com/minvws/nl-rdo-manon/blob/5c99b9974ae55056ce052679db681bad8ac5aef4/manon/footer-navigation-link-focus-variables.scss#L14-L18